books/All.md

```
@@ -8922,6 +8922,68 @@ class Solution:
         return res
 ```
 
+## 1431. Kids With the Greatest Number of Candies [Easy]
+https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)